### ISSUE TITLE
something is wrong with Shadow 

### ISSUE THREAD

**Issue by vis2021t** [abrahamjuliot/creepjs#191]
Buddy I for the first time in 2 month used my pc, I opened creepjs website it shows 10+ scores on shadow and 100% on trust score 

I left it and came back after a while then I refreshed the page and the shadow score decreased even though nothing was changed from end. 

Something is definitely wrong as I tried same with my father's phone which was never used in these stuff and it was same outcome in change score after refreshing at shadow section.

**Comment by abrahamjuliot** [comment]
On initial load, speech remote voices sometime fail to load (within the 3-second timeout). This is common on Android. I hope that's not it. But that, for example, would remove the bonus `10+`. I could remove remote voices from the fingerprint and skip on mobile devices, but there are a lot of factors like this that will cause small changes in the loose fingerprint. ideally, we want to reduce them where possible.

If that tab remains open, you can sometimes see these changes under revision in the browser section. I can add a decrypt option to the shadow fingerprint, and we can analyze areas that changed. I need to restructure the fuzzy hash better though and reset the fingerprints. Ideally, each shadow bin will represent similar values. Some bins could hold 1 value and some could hold more. I would put all screen metrics in 1 bin, for example, and cores and memory would have their own bin.

**Comment by abrahamjuliot** [comment]
This should be more stable now. I set a `50ms` delay before `getVoices` executes on the load event listener. From what I can tell, this gives the voice engines enough time to load, eliminating the random timeout on Android. In Chrome, I observed a few factors that could change the shadow fingerprint when switching to a guest window.

- Google's remote voices are absent
- The following are reset to the browser default:
  - permissions
  - do not track
  - languages
